docs/en/stack/ml/df-analytics/dfa-regression.asciidoc

```diff
@@ -71,14 +71,20 @@ the {depvar} by combining the predictions from multiple base learners –
 algorithms that generalize from the dataset. The performance of an ensemble is 
 usually better than the performance of each individual base learner because the 
 individual learners will make different errors. These average out when their 
-predictions are combined. The ensemble learning technique that we use in the 
-{stack} is a type of boosting called extreme gradient boost (XGboost) which 
-combines decision trees with gradient boosting methodologies.
+predictions are combined.
+
+[discrete]
+[[dfa-regression-model]]
+===== {regression-cap} models
```

Comment:
Is model the right term here?  In other places, we talk about creating models and model snapshots, so that's what I think of when I read the term. Whereas it seems to me this section is more about the algorithms that we use to create model.

```suggestion
===== {regression-cap} algorithms
```